contributor-guide/modules/ROOT/pages/testing/intro.adoc

```diff
@@ -36,6 +102,8 @@ Then, read the documentation for the following libraries, which support the writ
 
 Start small and develop a good understanding of how these testing libraries work, before writing more than a few tests.
 
+=== Advanced Testing
+
 When you have a good understanding of the basic testing procedures, look into more advanced techniques, such as xref:testing/fuzzing.adoc[] and xref:testing/continuous-integration.adoc[].
```

Comment:
> such as xref:testing/fuzzing.adoc[] and xref:testing/continuous-integration.adoc[].

`xref:testing/continuous-integration.adoc[]` seems out of place, because there's a "Test Matrix" section above and one can only set up the test matrix after setting up `xref:testing/continuous-integration.adoc[]`.

Maybe just `such as xref:testing/fuzzing.adoc[]` is OK here.